var/spack/repos/builtin/packages/py-amrex/package.py

```diff
@@ -113,7 +113,7 @@ def copy_test_sources(self):
         install test subdirectory for use during `spack test run`."""
         cache_extra_test_sources(self, [self.tests_src_dir])
 
-    def test(self):
+    def test_pytest(self):
         """Perform smoke tests on the installed package."""
         pytest = which("pytest")
         pytest(join_path(install_test_root(self), self.tests_src_dir))
```

Comment:
These test should not be run from under the installation prefix.  Is it possible to run them from the test stage directory?

Reply:
Yes, that works. Just needs to copy the tests directory.
https://github.com/AMReX-Codes/pyamrex/tree/development/tests

Reply:
That copy (or caching) of the directory is already being done at line 114.